repl/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>LibJS REPL</title>
+    <link rel="shortcut icon" type="image/png" href="/favicon.png" />
+    <link rel="stylesheet" href="main.css" />
+    <link rel="stylesheet" href="/test262/main.css" />
```

Comment:
_probably_ not needed, otherwise we should extract the common styles

Reply:
It uses the colour variables and main page styles still.

Reply:
The "shared" rules take up more than 80% of the whole file, sounds fine to just pull in like this, but I can try to split them into a separate file if you want

Reply:
nah, it's fine :)